pytest.ini

```diff
@@ -0,0 +1,4 @@
+[pytest]
+doctest_optionflags = NORMALIZE_WHITESPACE ELLIPSIS
+junit_family = xunit2
+addopts = --doctest-modules -s -vv --durations=0
```

Comment:
doesn't `-s -vv` make the logs too verbose? is `-v` enough?

Reply:
No I don't think so, Gael specifically wanted to see the output from the tests as they were run and we need -vv for that.

Reply:
ok no problem (BTW it is the `-s` option that tells pytest not to capture output)

Reply:
-s is included in the pytest.ini